backend/src/handler.py

```diff
@@ -61,13 +61,13 @@ def _get_instances_from_rules(event, context):
     parameters = event["queryStringParameters"]
     if parameters is None:
         parameters = {}
-    parameters.update({
-        'start': '',
-        'end': '',
-        'current': '',
-        'set_aside': '',
-        'biweekly_start': '',
-    })
+        parameters.update({
+            'start': '',
+            'end': '',
+            'current': '',
+            'set_aside': '',
+            'biweekly_start': '',
+        })
```

Comment:
??? Isn't this fine?

Reply:
When the "parameters.update" code is not indented under the "if parameters is none:" statement, it has the effect of setting parameter values provided by the user to ''.